demisto_sdk/commands/validate/file_validator.py

```diff
@@ -523,13 +527,121 @@ def validate_committed_files(self):
 
     def validate_pack_unique_files(self, packs):
         for pack in packs:
+            print(f'Validating {pack} unique pack files')
             pack_unique_files_validator = PackUniqueFilesValidator(pack)
             pack_errors = pack_unique_files_validator.validate_pack_unique_files()
             if pack_errors:
                 print_error(pack_errors)
                 self._is_valid = False
 
+    def file_all_validations(self, file_path, file_type: str = None):
+        file_extention = os.path.splitext(file_path)[-1]
+        if re.match(TEST_PLAYBOOK_REGEX, file_path, re.IGNORECASE) and not file_type:
+            return
+        if file_extention not in ['.yml', 'json', '.md']:
+            return
+        # Ignoring changelog and description files
+        if 'changelog' in file_path.lower() or 'description' in file_path.lower():
+            return
+        print(f'Validating {file_path}')
+        if re.match(TEST_PLAYBOOK_REGEX, file_path, re.IGNORECASE):
+            return
+
+        elif 'README' in file_path:
+            readme_validator = ReadMeValidator(file_path)
+            if not readme_validator.is_valid_file():
+                self._is_valid = False
+            return
+        elif 'description' in file_path.lower():
+            description_validator = DescriptionValidator(file_path)
+            if not description_validator.is_valid():
+                self._is_valid = False
+                return
+        structure_validator = StructureValidator(file_path, predefined_scheme=file_type)
+        if not structure_validator.is_valid_file():
+            self._is_valid = False
+
+        elif re.match(PLAYBOOK_REGEX, file_path, re.IGNORECASE) or file_type == 'playbook':
```

Comment:
what about pack/beta playbooks?

Reply:
beta playbooks will be removed to legacy playbooks

Packs playbooks are validated

Reply:
where are they validated?

Reply:
[this](https://github.com/demisto/demisto-sdk/blob/4093d6b60fab342418ae0c439de0a2bad9655c94/demisto_sdk/commands/validate/file_validator.py#L639) catches all files in Packs recursively

Reply:
but then you get here and ignore them since they are not accepted by this regex

Reply:
The regexes are broken and need to have a robust fix.
The playbooks are not ignored because of the `or file_type == 'playbook'` part